README.md

```diff
@@ -1,5 +1,6 @@
 # The Rust Programming Language
 
+[![CI](https://github.com/rust-lang/rust/actions/workflows/ci.yml/badge.svg)](https://github.com/rust-lang/rust/actions/workflows/ci.yml)
```

Comment:
IIRC, if no branch is specified, the CI status of the default branch (master in our case) will be referred, but we [do not run CI on the master branch](https://github.com/rust-lang/rust/actions/workflows/ci.yml?query=branch%3Amaster), so this may not reflect an actual CI status.